src/ResourceManager/ManagementGroups/Commands.ManagementGroups/packages.config

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.2" targetFramework="net452" />
```

Comment:
@rajshah11 similarly to your `.csproj` file, you can remove all of the common packages (it should be everything but the `Microsoft.Azure.Management.ResourceManager` package)

Reply:
Done.